src/ImageExif/ImageExif.js

```diff
@@ -0,0 +1,211 @@
+          const tagKeys = Object.keys(tags).sort()
+          const sortedProperties = {}
+
+          tagKeys.forEach(x => {
+            sortedProperties[x] = tags[x].description
+          })
+
+          // Image Location
+          const latRef = sortedProperties.GPSLatitudeRef === 'North latitude' ? 'N' : 'S'
+          const longRef = sortedProperties.GPSLongitudeRef === 'West longitude' ? 'W' : 'E'
+          const imageLat = (latRef === 'N') ? sortedProperties.GPSLatitude : sortedProperties.GPSLatitude * -1
+          const imageLong = (longRef === 'W') ? (sortedProperties.GPSLongitude * -1) : sortedProperties.GPSLongitude
+          const coords = fromLonLat([imageLong, imageLat])
+          const centerPoint = new Point(coords)
+          const imageLocationFeature = new Feature(centerPoint)
+
+          imageLocationFeature.setProperties(sortedProperties)
+          imageLocationFeature.set('title', 'Captured Image')
+          imageLocationFeature.set('name', imageName)
+          imageLocationFeature.set('imageId', imageId)
+          imageLocationFeature.set('imageURL', imageURL)
+          features.push(imageLocationFeature)
+
+          // Image Location Error Margin
+          const imageLocationError = Number(sortedProperties.GPSHPositioningError)
+
+          if (imageLocationError) {
+            const errorRadius = new Circle(coords, imageLocationError)
+            const errorRadiusFeature = new Feature(errorRadius)
+
+            errorRadiusFeature.setProperties({ 'Error Radius (meters)': imageLocationError })
+            errorRadiusFeature.set('title', 'Image Location Error Radius')
+            errorRadiusFeature.set('name', 'Error Radius')
+            errorRadiusFeature.set('imageId', imageId)
+            errorRadiusFeature.set('imageURL', imageURL)
+            errorRadiusFeature.setStyle(new Style({
+              stroke: new Stroke({
+                lineDash: [4, 8],
+                color: '#ff0000',
+                width: 2
+              })
+            }))
+            features.push(errorRadiusFeature)
+          }
+
+          // Image Bearing Line
+          const imageBearing = sortedProperties.GPSDestBearing
+
+          if (imageBearing) {
+            const geojsonPoint1 = olToGeojson(imageLocationFeature)
+            const geojsonPoint2 = transformTranslate(geojsonPoint1, 2, imageBearing, { units: 'meters' })
+            const olPoint1 = geojsonToOl(geojsonPoint1).getGeometry().getCoordinates()
+            const olPoint2 = geojsonToOl(geojsonPoint2).getGeometry().getCoordinates()
+            const lineThing = new LineString([olPoint1, olPoint2])
+            const bearingFeature = new Feature(lineThing)
+
+            bearingFeature.set('title', 'Image Bearing')
+            bearingFeature.set('name', 'Image Bearing')
+            bearingFeature.set('imageId', imageId)
+            bearingFeature.setStyle(styleFunction)
+            bearingFeature.set('imageURL', imageURL)
```

Comment:
is there a reason you're not spreading all of the exif attributes on the feature?

Reply:
I am on the `imageLocationFeature` (`imageLocationFeature.setProperties(sortedProperties)`), since it's the feature that actually represents the location/image, but I didn't on the other features since they're ancillary and only showing one aspect of the image (bearing, gps error margin).

Should I just put them all on everything you think?